hack/k8s/helm/nuclio/values.yaml

```diff
@@ -136,9 +136,13 @@ registry: {}
     # username: someuser
     # password: somepass
 
-  #  Use dedicated base/onbuild images registry (pull registry)
+  #  Use dedicated onbuild images registry (pull registry)
```

Comment:
```suggestion
  #  Use a dedicated "onbuild" images registry (pull registry)
```

Reply:
Perhaps also mention in the comment the alternative of using `dependantImageRegistryURL` for configuring an "onbuild" and base images pull registry — for example, add a period at the end of the existing comment and add this:
```
  # Note: To configure a pull registry for both "onbuild" and base images, use `dependantImageRegistryURL`.
```